### DISCUSSION TITLE
PEST-GLM versus PEST-HP

### DISCUSSION THREAD

**Discussion by sjepsen395** [usgs/pestpp#297] [General]
Hi All,

We are trying two algorithms to calibrate our model, the Iterative Ensemble Smoother (IES) in PEST++ and the Gauss-Levenberg-Marquardt (GLM) approach. For the GLM approach, would you generally recommend using PEST-GLM or PEST-HP? PEST-GLM seems to have more dynamic options for SVD assist whereas maybe PEST-HP is further along in its development? Just guessing. I think I read in another discussion that PEST-HP was the recommended option. We may want to use SVD assist because each model run takes about 10 minutes and not sure what the budget is for cloud computing.

Thanks,
Steve

**Comment by jtwhite79** [comment]
I would def use pest_hp for deterministic regularized parameter estimation - its much more battle hardened, especially for the regularization solve.  There are still the pest utilities to setup and implement the svd assist stuff if you want to use that.  

How many parameters do you have?

**Reply by sjepsen395** [reply]
Hi jtwhite79,

OK thanks. Good to know. I'll plan on going with PEST_HP for the deterministic approach to parameter estimation.

We have 668 parameters total. This could double in the near future (we are currently calibrating only the aquifer part of the model). A simple test run with 19 adjustable parameters took 24 hours on my local machine running 5 agents. A single model run takes about 10 minutes.

Thanks,
Steve

**Reply by jtwhite79** [reply]
I see.  5 agents is gonna be a lil slow...

**Reply by sjepsen395** [reply]
Yeah. IES is sounding pretty good right now. Thank you very much.
